Content.Shared/Medical/VomitActionComponent.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Robust.Shared.Containers;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Medical; // Kaif SYSTEM
+
+/// <summary>
+/// This is used for...
+/// </summary>
+[RegisterComponent]
+public sealed partial class VomitActionComponent : Component
+{
+    [DataField("vomitAction", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string? VomitAction = "ActionVomit";
+
+    [DataField("vomitActionEntity")]
+    public EntityUid? VomitActionEntity;
+
+    [DataField("thirstAdded")]
+    public float ThirstAdded = 40f;
+
+    [DataField("hungerAdded")]
+    public float HungerAdded = 40f;
+
+    public Container Stomach = default!;
+}
```

Comment:
```suggestion
[RegisterComponent, NetworkedComponent]
public sealed partial class VomitActionComponent : Component
{
    [DataField]
    public EntProtoId? VomitAction = "ActionVomit";

    [DataField]
    public EntityUid? VomitActionEntity;

    [DataField]
    public float ThirstAdded = 40f;

    [DataField]
    public float HungerAdded = 40f;

    public Container Stomach = default!;
}
```

Не уверен, нужно ли VomitAction делать нуллабл, у него же есть значение дефолтное.